src/index.html

```diff
@@ -30,6 +30,16 @@ <h1 id="logo-text" class="text-center">
         </div>
       </header>
       <section>
+        <form name="myForm" class="form">
```

Comment:
This name can be something like "librarySwitchForm".

Reply:
Fixed in: 91555c04c8088ee472c87cc25843642994ec92b4